pinc/bootstrap.inc

```diff
@@ -27,6 +27,9 @@ include_once($relPath.'site_vars.php');
 // Register autoloader
 spl_autoload_register('dp_class_autoloader');
 
+// Load composer autoloader
+require_once($relPath.'../vendor/autoload.php');
```

Comment:
this is ok on all pages even if they don't use composer deps?

Reply:
Yup. This just loads the autoloader so that when classes are used the autoloader finds the right files to load. The docs on the [autoloader](https://www.php.net/manual/en/language.oop5.autoload.php) for more information on how this works..

Reply:
Nice, i learn something every day